onnxconverter_common/onnx_ops.py

```diff
@@ -66,7 +66,10 @@ def _apply_pointwise_operation(scope, op_type, input_names, output_name, contain
     elif container.target_opset < 8:
         op_version = 6
     else:
-        op_version = 8
+        if container.target_opset < 12 or op_type == 'Mean':
```

Comment:
Why does this 'mean' have to be 8?  or should we create a apply_mean for that?

Reply:
Mean has common logic as Max and Min for opset < 12. For opset 12, only Max and Min has the opset 12 version to support int, Mean does not change.

Reply:
The latest Mean's opset is opset 8.